keps/sig-network/1111-dynamically-re-configure-lb-node-set/kep.yaml

```diff
@@ -0,0 +1,23 @@
+id: ""
+prnumber: ""
+name: 1111-dynamically-re-configure-lb-node-set
+title: Dynamically re-configure LB node set
+kep-number: "1111"
+authors: ['@alexanderConstantinescu']
+owning-sig: sig-network
```

Comment:
should we add sig-cloud-provider ?
/cc @andrewsykim

Reply:
We could, but maybe @nckturner since he commented on the original issue I filed: https://github.com/kubernetes/kubernetes/issues/111539 ?